### PR TITLE
Make gce_safe_format_and_mount actually work correctly.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by zmerlynn** [comment]
LGTM.

File: pkg/kubelet/volume/gce_pd/gce_pd.go [hunk 1]
```diff
@@ -89,16 +90,17 @@ func (plugin *gcePersistentDiskPlugin) newBuilderInternal(spec *api.Volume, podU
...
-		legacyMode: false,
+		podUID:      podUID,
+		volName:     spec.Name,
+		pdName:      pdName,
+		fsType:      fsType,
+		partition:   partition,
+		readOnly:    readOnly,
+		manager:     manager,
+		mounter:     mounter,
+		diskMounter: &gceSafeFormatAndMount{mounter, exec.New()},
```
Comment (thockin) [inline pkg/kubelet/volume/gce_pd/gce_pd.go]: I don't see where this is ever used?

**Comment by brendandburns** [comment]
ugh, you're right... *sigh* it's friday...

I'll send another PR tonight.

**Comment by thockin** [comment]
Maybe changes like this should actually be run for real once before merge :)

**Comment by brendandburns** [comment]
It was run.  It "works" correctly, it's just a no-op...  So it's kind of
hard to validate.

But yes, now that it's in, I'm going to update pd.sh to validate the
behavior.

--brendan

**Pull request merged**